update doc related to fast-create (#18517)
pingcap/docs: upgrade-tidb-using-tiup.md

> 3. **DO NOT** upgrade a TiDB cluster when a DDL statement is being executed in the cluster (usually for the time-consuming DDL statements such as `ADD INDEX` and the column type changes). Before the upgrade, it is recommended to use the [`ADMIN SHOW DDL`](/sql-statements/sql-statement-admin-show-ddl.md) command to check whether the TiDB cluster has an ongoing DDL job. If the cluster has a DDL job, to upgrade the cluster, wait until the DDL execution is finished or use the [`ADMIN CANCEL DDL`](/sql-statements/sql-statement-admin-cancel-ddl.md) command to cancel the DDL job before you upgrade the cluster.
> 4. If the TiDB version before upgrade is 7.1.0 or later, you can ignore the preceding warnings 2 and 3. For more information, see [limitations on using TiDB smooth upgrade](/smooth-upgrade-tidb.md#limitations).
> 5. Be sure to read [limitations on user operations](/smooth-upgrade-tidb.md#limitations-on-user-operations) before upgrading your TiDB cluster using TiUP.

> > 6. If the current cluster version is from TiDB v7.6 to v8.2, the upgrade target version is v8.3.0 or later, and [accelerated table creation](/accelerated-table-creation.md) is enabled, you need to disable this feature first, and then enable it as needed after the upgrade is completed. Otherwise, some metadata KVs added by this feature will remain in the cluster. Starting from v8.3.0, this feature has been optimized. Upgrading to a higher version will no longer generate and retain this type of metadata KVs.

Author: ```suggestion
> 6. If the current cluster version is from TiDB v7.6.0 to v8.2.0, the upgrade target version is v8.3.0 or later, and [accelerated table creation](/accelerated-table-creation.md) is enabled, you need to disable this feature first, and then enable it as needed after the upgrade is completed. Otherwise, some metadata KVs added by this feature will remain in the cluster. Starting from v8.3.0, this feature has been optimized. Upgrading to a higher version will no longer generate and retain this type of metadata KVs.
```

> > 6. If the current cluster version is from TiDB v7.6.0 to v8.2.0, the upgrade target version is v8.3.0 or later, and [accelerated table creation](/accelerated-table-creation.md) is enabled, you need to disable this feature first, and then enable it as needed after the upgrade is completed. Otherwise, some metadata KVs added by this feature will remain in the cluster. Starting from v8.3.0, this feature has been optimized. Upgrading to a higher version will no longer generate and retain this type of metadata KVs.

Reviewer: ```suggestion
> 6. If the current cluster version is from TiDB v7.6.0 to v8.2.0, the upgrade target version is v8.3.0 or later, and [`tidb_enable_fast_create_table`](/system-variables.md#tidb_enable_fast_create_table-new-in-v800) is enabled, you need to disable this feature first, and then enable it as needed after the upgrade is completed. Otherwise, some metadata KVs added by this feature will remain in the cluster. Starting from v8.3.0, this feature has been optimized. Upgrading to a higher version will no longer generate and retain this type of metadata KVs.
```

> > 6. If the current cluster version is from TiDB v7.6.0 to v8.2.0, the upgrade target version is v8.3.0 or later, and [`tidb_enable_fast_create_table`](/system-variables.md#tidb_enable_fast_create_table-new-in-v800) is enabled, you need to disable this feature first, and then enable it as needed after the upgrade is completed. Otherwise, some metadata KVs added by this feature will remain in the cluster. Starting from v8.3.0, this feature has been optimized. Upgrading to a higher version will no longer generate and retain this type of metadata KVs.

Reviewer 2: ```suggestion
> 6. If the version of your current TiDB cluster is TiDB v7.6.0 to v8.2.0, the upgrade target version is v8.3.0 or later, and accelerated table creation is enabled via [`tidb_enable_fast_create_table`](/system-variables.md#tidb_enable_fast_create_table-new-in-v800), you need to first disable the accelerated table creation feature, and then enable it as needed after the upgrade is completed. Otherwise, some metadata KVs added by this feature remain in the cluster. Starting from v8.3.0, this feature is optimized. Upgrading to a later TiDB version no longer generates and retains this type of metadata KVs.
```
